build-tools/packages/build-cli/README.md

```diff
@@ -48,6 +48,7 @@ USAGE
 * [`flub run`](docs/run.md) - Generate a report from input bundle stats collected through the collect bundleStats command.
 * [`flub test-only-filter`](docs/test-only-filter.md) - FOR INTERNAL TESTING ONLY. This command is used only to test the common package filtering and selection logic that is used across the CLI. FOR INTERNAL TESTING ONLY.
 * [`flub typetests`](docs/typetests.md) - Updates configuration for type tests in package.json files. If the previous version changes after running preparation, then npm install must be run before building.
+* [`flub updates`](docs/updates.md) - Updates the types in package.json based on the release flag
```

Comment:
We have a lot of flub commands that are intended to be used by developers. I belive this one will only ever be used by publishing piplelines, so I think it might be good to note that in the documention here.

Also we have several things flub can update already, (like type test configuration, dependencies of packages, assert tags, etc), so I think the name "update" is way too general. Also this isn't really "updating" the value (like may of flubs other commands, like updating depdencies), ots more of a "change".

Maybe naming it "setReleaseTagForPublishing" would solve both issues. And since we don't expect people to have to ever invoke this command manually (except for when you are testing it out), having a long name shouldn't be too bad (and I think flub support tab completion anyway if you set it up)

Reply:
I agree with Craig. Maybe `update tsdocs` or `update api-extractor-output`?

Reply:
set-api-extractor-settings?